ports/espressif/boards/waveshare_esp32_s3_zero/mpconfigboard.mk

```diff
@@ -0,0 +1,20 @@
+USB_VID = 0x303a
+USB_PID = 0x1fe8
```

Comment:
Where does this PID come from? I don't see it listed here: https://github.com/espressif/usb-pids/blob/main/allocated-pids.txt

Reply:
_We discussed on github but I'll mention it here for the sake of completeness. I got it from running lsusb under Linux. I think this was after I already put circuitpython on it though so very likely not what it should be_

I've contacted Waveshare to see if they want to request a PID from espressif or if I can/should on their behalf. No response so far.